### DISCUSSION TITLE
Changing FormBuilderFieldOption text color

### DISCUSSION THREAD

**Discussion by JIFT** [danvick/flutter_form_builder#1006] [Q&A]
HI all, I am struggling since 2 days ago with this component. I am trying to use a FormBuilderChoiceChip and the idea is that when a FormBuilderFieldOption is selected, I want to change the background color and text color. The problem is that the selectedColor property just change the background, not the text color. Is there a way to achieve this?

**Comment by deandreamatias** [comment]
This still a question?
